ann/package.json

```diff
@@ -3,49 +3,33 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "nodemon -L ./server/server.mjs",
+    "dev": "next dev",
```

Comment:
Moved back to using `next dev`. Not sure why it was changed, but you definitely should be using `next dev`.